scripts/autoshutdown.cron.sh

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env bash
+
+# This script is intended to be called once per minute. If it doesn't detect
```

Comment:
`check && sleep 10m && check && shutdown -h now` is not great, you are logged in for a few minutes, then out for a few minutes etc it's likely going to hit.

I do understand what you mean by the coupling. I am changing it to take the number of iterations as an argument, so at least it's all on the calling side.